main.tf

```
@@ -16,6 +16,7 @@ module "lacework_cfg_iam_role" {
   version                 = "~> 0.2"
```

This change actually depends on https://github.com/lacework/terraform-aws-iam-role/pull/32

We need to bump the min version to:
```suggestion
  version                 = "~> 0.3"
```